openmc/tallies.py

```diff
@@ -457,6 +459,75 @@ def sparse(self, sparse):
                 self._std_dev = np.reshape(self._std_dev.toarray(), self.shape)
             self._sparse = False
 
+    def write_to_vtk(self, filename):
+        """Writes the tally to a vtk file
+
+        Args:
+            filename (str): the filename (must end with .vtk)
+
+        Raises:
+            ValueError: if no MeshFilter with appropriate mesh was found 
+                (SphericalMesh not supported)
+        """
+        # check that tally has a MeshFilter
+        mesh = None
```

Comment:
Perhaps something like this

```suggestion
        try:
                import vtk
                import vtk.util.numpy_support as nps
        except ImportError:
                msg = 'Python package vtk was not found, please install vtk to use Tally.write_to_vtk.'
                raise ImportError(msg)
        mesh = None
```